repose-aggregator/components/filters/rate-limiting-filter/src/main/java/org/openrepose/filters/ratelimiting/RateLimitingHandler.java

```diff
@@ -42,16 +42,22 @@
 import java.io.ByteArrayOutputStream;
 import java.io.InputStream;
 import java.nio.charset.StandardCharsets;
-import java.util.List;
-import java.util.Optional;
+import java.util.*;
 import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 
 import static javax.servlet.http.HttpServletResponse.*;
 
 /* Responsible for handling requests and responses to rate limiting, also tracks and provides limits */
 public class RateLimitingHandler {
 
     private static final Logger LOG = org.slf4j.LoggerFactory.getLogger(RateLimitingHandler.class);
+    private static final Set<MimeType> SUPPORTED_MIME_TYPES = Stream.of(
```

Comment:
```
com.google.common.collect.Sets.newHashSet(
    MimeType.APPLICATION_JSON,
    MimeType.APPLICATION_XML);
```
or
```
com.google.common.collect.ImmutableSet.of(
    MimeType.APPLICATION_JSON,
    MimeType.APPLICATION_XML);
```

Both would require adding `com.google.guava:guava` to the filter's dependencies, though.